### ISSUE TITLE
Installation fails with error

### ISSUE THREAD

**Issue by preetb123** [react-native-community/react-native-template-typescript#106]
# Bug

```

info Setting up new React Native app in /Users/preetam/projects/MyApp
info Fetching template react-native-template-react-native-template-typescript...
yarn add v1.12.3
[1/4] 🔍  Resolving packages...
warning jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > left-pad@1.3.0: use String.prototype.padStart()
error An unexpected error occurred: "https://registry.yarnpkg.com/react-native-template-react-native-template-typescript: Not found".
info If you think this is a bug, please open a bug report with the information provided in "/Users/preetam/projects/MyApp/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
yarn remove v1.12.3
[1/2] 🗑  Removing module react-native-template-react-native-template-typescript...
error This module isn't specified in a package.json file.
info Visit https://yarnpkg.com/en/docs/cli/remove for documentation about this command.
warn Failed to clean up template temp files in node_modules/react-native-template-react-native-template-typescript. This is not a critical error, you can work on your app.
(node:77612) UnhandledPromiseRejectionWarning: Error: Command failed: yarn add react-native-template-react-native-template-typescript
    at makeError (/Users/preetam/projects/MyApp/node_modules/execa/index.js:174:9)
    at /Users/preetam/projects/MyApp/node_modules/execa/index.js:278:16
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async createFromRemoteTemplate (/Users/preetam/projects/MyApp/node_modules/@react-native-community/cli/build/tools/generator/templates.js:115:5)
    at async createProjectFromTemplate (/Users/preetam/projects/MyApp/node_modules/@react-native-community/cli/build/tools/generator/templates.js:88:3)
    at async generateProject (/Users/preetam/projects/MyApp/node_modules/@react-native-community/cli/build/commands/init/initCompat.js:116:3)
    at async Object.initCompat (/Users/preetam/projects/MyApp/node_modules/@react-native-community/cli/build/commands/init/initCompat.js:103:3)
(node:77612) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:77612) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```
## Environment info

React native info output:

```bash
System:
    OS: macOS Mojave 10.14.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 252.18 MB / 16.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.12.0 - /usr/local/bin/node
    Yarn: 1.12.3 - /usr/local/bin/yarn
    npm: 6.11.3 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.0, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
    Android SDK:
      API Levels: 23, 26, 27, 28
      Build Tools: 25.0.0, 26.0.1, 26.0.3, 27.0.3, 28.0.2, 28.0.3
      System Images: android-26 | Google APIs Intel x86 Atom_64
  IDEs:
    Android Studio: 3.1 AI-173.4907809
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react-native: 0.61.4 => 0.61.4 
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

**Comment by MateusAndrade** [comment]
You can try running: ``` npx react-native init MyApp --typescript```?

**Comment by radko93** [comment]
Duplicate of #80

**Comment by ollydixon** [comment]
Someone needs to update this page, https://reactnative.dev/docs/typescript
